doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1615,6 +1634,16 @@ The view is applied to the top assumption.
 
   a first ``move=> *`` adds only ``_a_ : bool`` and ``_b_ : bool``
   to the context; it takes a second ``move=> *`` to add ``_Hyp_ : _a_ = _b_``.
+``+``
```

Comment:
This one is cool :grin:

Reply:
oops

Reply:
I didn't mean to trigger this reaction. I just really like `move => + x`

Reply:
I mean "oops, I forgot to document htis". No problem.

Reply:
Sorry I got confused, I thought the lower diff was glued, hence that there was no line for documenting `+`.